packages/noya-renderer/src/components/DragHandles.tsx

```diff
@@ -13,9 +23,30 @@ interface Props {
 
 export default memo(function DragHandles({ selectionPaint, rect }: Props) {
   const CanvasKit = useCanvasKit();
+  const [state] = useApplicationState();
 
   const dragHandlePaint = useColorFill(CanvasKit.Color(255, 255, 255, 1));
-  const dragHandles = getDragHandles(rect);
+
+  let dragHandles = getDragHandles(rect);
```

Comment:
Maybe rename `getDragHandles` => `getRectDragHandles` for consistency with the line version